src/deploy/functions/deploymentPlanner.ts

```diff
@@ -34,7 +34,6 @@ export interface RegionMap {
 export interface RegionalDeployment {
   region: string;
   sourceToken?: string;
-  firstFunctionDeployment?: () => any;
```

Comment:
Realized that I could turn this into an implementation detail of runRegionalFunctionDeployment, so I removed this from the type.